.varci.yml

```diff
@@ -0,0 +1,8 @@
+ruleset:   
+  tag_support:    
+    name: "Label issues as support"   
+    events: [ issues ]    
+    label: support    
+    when:   
+      - body contains "/\[\s?x\s?\] support/"
```

Comment:
Spotted an omission @jadjoubran. `contains` should have been switched to `matches` when I changed it to regex. Want me to submit new PR or you'll do it?